app/views/govuk_component/tasklist.raw.html.erb

```diff
@@ -0,0 +1,84 @@
+<%
+  steps ||= false
+  small ||= false
+  heading_level ||= 2
+  open_section ||= false
+  remember_last_section ||= false
+
+  section_count = 0
+%>
+<% if steps %>
+  <div
```

Comment:
This whole file is hugely nested. Potentially consider breaking up into smaller partials.

Reply:
It is a bit. I'll try some extra line spaces first to see if that helps.